actions/views/channel.js

```diff
@@ -183,22 +193,30 @@ export function increasePostVisibility(channelId, beforePostId) {
             data: true,
             channelId,
         });
-
+        const time = Date.now();
         const result = await dispatch(PostActions.getPostsBefore(channelId, beforePostId, 0, Posts.POST_CHUNK_SIZE / 2));
         const posts = result.data;
 
-        const actions = [{
+        let actions = [{
```

Comment:
no need to change this to `let`

Reply:
Changed to const but i made a minor change to this part. `increasePostListVisiblity` is called when loading more posts and we don't need to save time then. 
It might save some bandwidth but it can cause some rare race conditions if like if socketConnects after user trying to load more posts etc. Extremely rare but nevertheless we don't need to have it